pkg/debug/checkups/processes.go

```diff
@@ -56,6 +57,15 @@ func (c *Processes) Run(ctx context.Context, fullWriter io.Writer) error {
 		if strings.Contains(strings.ToLower(exe), "kolide") {
 			c.kolideCount += 1
 			c.data[fmt.Sprintf("%d", p.Pid)] = pMap
+
+			if c.kolideRunningAsRoot {
+				continue
+			}
+
+			username := pMap["username"].(string)
+			if username == "root" || username == "NT AUTHORITY\\SYSTEM" {
+				c.kolideRunningAsRoot = true
+			}
```

Comment:
```suggestion

			if !c.kolideRunningAsRoot {
				username := pMap["username"].(string)
				if username == "root" || username == "NT AUTHORITY\\SYSTEM" {
					c.kolideRunningAsRoot = true
				}
			}
```

In this case, I think _not_ early return is more clear.

Reply:
agree, updated